src/tests/hook.test.ts

```diff
@@ -3,11 +3,12 @@ import {
 } from '../requestHandlers/hook';
 import * as hookService from '../services/hookService';
 import * as panelService from '../services/panelService';
-import * as panelSetService from '../services/panelSetService';
+import * as panelSet from '../requestHandlers/panelSet';
 import { Sequelize } from 'sequelize';
 jest.mock('../services/hookService');
 jest.mock('../services/panelService');
 jest.mock('../services/panelSetService');
+jest.mock('../requestHandlers/panelSet');
```

Comment:
If I'm understanding correctly, we are mocking the request handler to get around the unit tests bug? I don't think we should be doing this, even if the unit tests failed. Since we are only testing the controllers, only services and other controllers should be tested. Controllers shouldn't depend on know about handlers. If that means we need to rewrite or discard the unit tests, I think that's a better option

Reply:
ok, so I should just let the 3 tests for hook fail?